contracts/src/v0.8/dev/interfaces/AuthorizedReceiverInterface.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface AuthorizedReceiverInterface {
```

Comment:
Naming is a little confusing to me - "Receiver" in the interface name but "Sender" in all method names.

Reply:
Because this interface will be used by other contracts like Operator.sol, I would not change naming.